ietf/name/fixtures/names.json

```diff
@@ -14042,7 +14042,7 @@
   },
   {
     "fields": {
-      "desc": "Legacy stream",
+      "desc": "Legacy",
```

Comment:
This won't actually change production. it will show up here as a side-effect of the change in the database instead. The PR will need a data migration to change the name.

Reply:
Got it.  It will take me a few days to re-understand the migration stuff. If you want to add a commit or hints to me, feel free.  (Not asking, just not wanting to be a blocker if I don't have to be)

Reply:
No worries, this can wait awhile